src/coreComponents/finiteElement/elementFormulations/H1_Tetrahedron_Lagrange1_Gauss.hpp

```diff
@@ -322,7 +323,199 @@ class H1_Tetrahedron_Lagrange1_Gauss1 final : public FiniteElementBase
   constexpr static real64 parentVolume = 1.0 / 6.0;
 
   /// The weight of each quadrature point.
-  constexpr static real64 weight = parentVolume / numQuadraturePoints;
+  constexpr static real64 weight = parentVolume;
+  //constexpr static real64 weight = parentVolume / numQuadraturePoints;
+
+  /**
+   * @brief Calculate the weights
+   * @param q
+   * @return weight.
+   */
+  GEOS_HOST_DEVICE
+  inline
+  constexpr static real64 quadratureWeight( localIndex const q )
+  {
+    if constexpr (NUM_Q_POINTS == 1)
+    {
+      real64 const w[numQuadraturePoints] = { 1.0 };
+      return w[q];
+    }
+    else if constexpr (NUM_Q_POINTS == 5)
+    {
+      real64 const w[numQuadraturePoints] = {-4.0/5.0, 9.0/20.0, 9.0/20.0, 9.0/20.0, 9.0/20.0 };
+      return w[q];
+    }
+    else if constexpr (NUM_Q_POINTS == 14)
+    {
+      real64 const w[numQuadraturePoints] = { 0.073493043116361949544,
+                                              0.073493043116361949544,
+                                              0.073493043116361949544,
+                                              0.073493043116361949544,
+                                              0.11268792571801585080,
+                                              0.11268792571801585080,
+                                              0.11268792571801585080,
+                                              0.11268792571801585080,
+                                              0.042546020777081466438,
+                                              0.042546020777081466438,
+                                              0.042546020777081466438,
+                                              0.042546020777081466438,
+                                              0.042546020777081466438,
+                                              0.042546020777081466438 };
+      return w[q];
+    }
+    else
+    {
+      GEOS_UNUSED_VAR( q );
+      GEOS_ERROR( "NUM_Q_POINTS not available for this element type" );
```

Comment:
maybe `static_assert(true, "NUM_Q_POINTS not available for this element type");` ?